OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -64,9 +64,9 @@ public void InvokeCommand(string name, string arg)
 			if (world.LocalPlayer == null)
 				return;
 
-			if (!developerMode.Enabled)
+			if (!developerMode.Enabled || world.LocalPlayer.WinState != WinState.Undefined)
 			{
-				Game.Debug("Cheats are disabled.");
+				Game.Debug("This command ({0}) is disabled.".F(name));
```

Comment:
I would prefer it if we could use "Developer commands are disabled.".  Just giving the command name with no extra context may be confusing.